scripts/capture.py

```diff
@@ -0,0 +1,31 @@
+import cv2
+import datetime
+import os
+
+OUTPUT_DIR = '/home/pi/data'
+DIR = '%Y-%m-%d'
+FILE = '%H:%M:%S.png'
+
+cap = cv2.VideoCapture(0)
+
+#  warmup camera
+for i in range(15):
+    _, _ = cap.read()
+
+ret, frame = cap.read()
+grayimg = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
+
+_, thresh = cv2.threshold(grayimg, 80, 255, cv2.THRESH_BINARY)
+_, thresh_inv = cv2.threshold(grayimg, 80, 255, cv2.THRESH_BINARY_INV)
+nonzero = cv2.countNonZero(thresh)
+nonzero_inv = cv2.countNonZero(thresh_inv)
+
+if nonzero - nonzero_inv > 0:
```

Comment:
@Foxiy what is your view on this? I am not convinced this is the best approach and believe that we should discuss it further.